internal/appsec/waf.go

```diff
@@ -277,6 +297,7 @@ const (
 	serverRequestPathParams           = "server.request.path_params"
 	serverRequestBody                 = "server.request.body"
 	serverResponseStatusAddr          = "server.response.status"
+	httpClientIP                      = "http.client_ip"
```

Comment:
Missing `Addr` suffix and the client ip address was badly named in the RFC so I'd suggest to hide it here and follow the same naming convention regardless (no strong opinion on this point):

```suggestion
	serverRequestPathParamsAddr       = "server.request.path_params"
	serverRequestBodyAddr             = "server.request.body"
	serverResponseStatusAddr      = "server.response.status"
	serverRequestClientIPAddr                  = "http.client_ip"
```
(it has formating issues I think)